kinesis/consumer.py

```
@@ -377,6 +433,136 @@ async def get_records(self, shard):
             # Connection or other issue
             return None
 
+    async def refresh_shards(self):
```

Pylint (3.3.7)

[refactor] Too many local variables (18/15)

(R0914)